source/develop/release-management/features/storage/qcow2v3.html.md

```diff
@@ -0,0 +1,73 @@
+---
+title: qcow2v3 image format
+category: feature
+authors: mlipchuk
+feature_name: qcow2v3 image format
+feature_modules: engine,vdsm
+feature_status: Implementation
+---
+
+# QCOW2V3 Image Format
+
+
+## Overview
+
+The QCOW image format includes a version number to allow introducing new features that change the image format in an incompatible way.
+Newer QEMU versions support a new format of QCow2 version 3, which is not backwards compatible and introduce
+many improvements like zero clusters, and performance improvement.
+That is the reason we enabled this new compatibility version for QCOW volumes in oVirt.
+
+## Owner
+
+- Name: [Maor Lipchuk](https://github.com/maorlipchuk)
+- Email: <mlipchuk@redhat.com>
+
+
+## Detailed Description
+
+The QCOW version 3 supports a 1.1 volume compatibility level.
+The QCOW version specifies to the qemu which qemu version the volume supports.
+The volume compatibility version is determined by the storage domain's version which the disk is created on.
+Storage domains with version lower than V4 support QCOW2 volumes, while V4 storage domains also support QCOW2v3.
+For more information about features of the different QCOW versions, see http://wiki.qemu.org/Features/Qcow3[here]
+
+### Installation/Upgrade
+
+As described before only 4.1 DC supports V4 storage domains.
+Once the Data Center is upgraded to 4.1 all the storage domains' version in this Data Center should be updated to V4,
+and every new storage domain will be created with V4 version.
+
+After the upgrade all new created QCOW volumes will be created with 1.1 compatibility level,
+while all the existing QCOW volumes that were created before the upgrade will be un changed regarding their compatibility level.
```

Comment:
I agree, wanted to merge the amend patches first and then describe the REST part, will add it also now that is merged.